tests/other/test_split_module.post.js

```diff
@@ -7,6 +7,9 @@ function saveProfileData() {
     var profile_data = new Uint8Array(buffer, offset, len);
     nodeFS.writeFileSync('profile.data', profile_data);
   }
+
+  // Say hello *after* recording the profile so that all functions are deferred.
+  Module['asm'].say_hello();
```

Comment:
```suggestion
  _say_hello();
```

Reply:
I updated the test to use malloc/free to allocate the buffer and I applied your suggested usage there as well.